scripts/src/report/get_verify_params.py

```diff
@@ -2,84 +2,64 @@
 import os
 import sys
 
+from submission import submission, validate
+
 sys.path.append("../")
 from chartprreview import chartprreview
 from signedchart import signedchart
 from tools import gitutils
 
 
-def get_report_full_path(category, organization, chart, version):
-    return os.path.join(
-        os.getcwd(), get_report_relative_path(category, organization, chart, version)
-    )
-
-
-def get_report_relative_path(category, organization, chart, version):
-    return os.path.join("charts", category, organization, chart, version, "report.yaml")
+def generate_verify_options(directory: str, s: submission.Submission):
+    """Prepare a set of GitHub outputs, including the options for chart-verifier verify
 
+    Args:
+        directory (str): Local directory in which to write the error logs
+        s (Submission): Submitted Github Pull Request
```

Comment:
NIT, feel free to ignore.

```suggestion
        s (Submission): Submitted GitHub Pull Request
```

Reply:
Done